mne/preprocessing/xdawn.py

```diff
@@ -62,6 +69,12 @@ def _least_square_evoked(epochs_data, events, tmin, sfreq):
         An concatenated array of toeplitz matrix for each event type.
     """
 
+    if isinstance(solver, string_types):
+        if solver == "pinv":
+            from ..stats.regression import _pinv_solver as solver
+        elif solver == 'cholesky':  # noqa
```

Comment:
Cause I'm stupid. I wanted to avoid this being counted for coverage, I'll remove it. 

I don't get what lines are. It being checked/why I get these coverage drops?